src/trading/base.py

```
@@ -66,42 +136,260 @@ def to_dict(self) -> dict[str, str]:
 
-class Trader(ABC):
-    """Base interface for trading operations."""
 
-    @abstractmethod
-    async def execute(self, *args, **kwargs) -> TradeResult:
-        """Execute trading operation.
-
-        Returns:
-            TradeResult with operation outcome
-        """
+def create_pump_fun_token_info(
```

Ruff (0.12.2)

Too many arguments in function definition (9 > 5)

(PLR0913)